pkg/api/metrics.go

```diff
@@ -61,5 +62,17 @@ func ReadMetrics(nodeExporterOutput []byte, loadPrefix string) (m Metrics, err e
 		return
 	}
 
+	availableMem, err := getField(loadPrefix+"memory_available_bytes", "")
+	if err != nil {
+		return
+	}
+	totalMem, err := getField(loadPrefix+"memory_total_bytes", "")
+	if err != nil {
+		return
+	}
+
+	// Subract off an extra 100 MiB to account for the kernel
+	m.MemoryUsageBytes = totalMem - availableMem - 100*(1<<20)
```

Comment:
The accounting here (i.e. `totalMem - availableMem`) is going to be accurate without the extra 100 MiB. The place where the kernel comes in is that `totalMem` is about 100 MiB less than it should be.

So `m.MemoryUsageBytes` should be `totalMem - availableMem + 100*(1<<20)`, afaict?

Alternatively, just using `availableMem` might be cleaner — I'm not super sure either way.

Reply:
Ahh I see - then I agree: it should be `+100*(1<<20)`